browser/js/common/factories/user/user.factory.js

```diff
@@ -22,6 +22,10 @@ app.factory('UserFactory', function ($http) {
 		},
 		deleteUser: function (id) {
 			return $http.delete('/api/user/' + id);
+		},
+		getLoggedInUser: function () {
+			return $http.get('/api/user/info')
+			.then(returnData);
 		}
```

Comment:
new function on UserFactory to get a logged in User. Created a new route to access this.